cmd/oc/oc.go

```diff
@@ -57,18 +43,9 @@ func main() {
 
 	// the kubectl scheme expects to have all the recognizable external types it needs to consume.  Install those here.
 	// We can't use the "normal" scheme because apply will use that to build stategic merge patches on CustomResources
-	utilruntime.Must(apps.Install(scheme.Scheme))
-	utilruntime.Must(authorization.Install(scheme.Scheme))
-	utilruntime.Must(build.Install(scheme.Scheme))
-	utilruntime.Must(image.Install(scheme.Scheme))
-	utilruntime.Must(oauth.Install(scheme.Scheme))
-	utilruntime.Must(project.Install(scheme.Scheme))
-	utilruntime.Must(installNonCRDQuota(scheme.Scheme))
-	utilruntime.Must(route.Install(scheme.Scheme))
-	utilruntime.Must(installNonCRDSecurity(scheme.Scheme))
-	utilruntime.Must(template.Install(scheme.Scheme))
-	utilruntime.Must(user.Install(scheme.Scheme))
-	legacy.InstallExternalLegacyAll(scheme.Scheme)
+	schemehelper.InstallSchemes(scheme.Scheme)
+	newapp.InstallSchemes()
```

Comment:
I added in here just for the readability because we need to warn code reader about new-app and set commands don't use global scheme. But I can also move these into their own init functions and can add a comment about this. WDYT?

Reply:
ah, you mean in a case somebody wants to modify the `schemehelper.InstallSchemes` functions? Ack, I think code-wise it would be better to move, and we can document this in all the places necessary.